tests/components/samsungtv/test_config_flow.py

```diff
@@ -70,7 +71,7 @@ def remote_fixture():
         yield remote
 
 
-async def test_user(hass, remote):
+async def test_user(hass: HomeAssistant, remote):
```

Comment:
What's the point of typing a single argument? Please either type the whole function or don't do it at all.